web/admin/src/Admin.js

```diff
@@ -27,7 +27,9 @@ export default class Admin extends PureComponent {
     sessions: {},
     questionsBySession: {},
     users: {},
-    publicSessions: {}
+    publicSessions: {},
+    helpText: "Add New Question",
```

Comment:
This `helpText` state variable seems like it has lots of places to end up in the wrong state, because the value it should have can be derived from other state variables, namely the existing questions and whether or not they are complete.  Any time you are trying to keep two pieces of state in sync, it is worth asking whether the render method can simply use one piece of state and derive the other.

Reply:
Would you say the main issue is how resethelptext is called?

Reply:
I would remove `resetHelpText()` and the `helpText` state altogether.  When it comes time to render, you can check the current questions to determine whether you want to show any help text.